config/config/sc-config.yaml

```diff
@@ -290,6 +302,126 @@ dex:
     # For more details, see https://elastisys.com/elastisys-engineering-how-to-use-dex-with-google-accounts-to-manage-access-in-kubernetes/
     # SASecretName: set-me
 
+thanos:
+  query:
+    # Enables the query/query-frontend components
+    enabled: true
+
+    # Number of query pods
+    # They are infinitly scalable
+    replicaCount: 1
```

Comment:
So this is used for both query and query frontend nr of replicas?

Reply:
Nope, this is just for the query pods. I think that the frontend pods shouldn't need to be scaled that much.

Reply:
Okay sure.